c2corg_ui/templates/waypoint/helpers/view.html

```diff
@@ -423,3 +423,27 @@
     </div>
   % endif
 </%def>
+
+<%def name="get_waypoint_access_field(locale, waypoint_type, open_tab=False)">\
+  % if locale.get('access'):
+    <%
+        if waypoint_type == 'access':
+            attribute_name = 'pt access'
+        elif waypoint_type in ['hut', 'climbing_indoor', 'climbing_outdoor']:
+            attribute_name = 'pedestrian access'
+        else:
+            attribute_name = 'road or pedestrian access'
```

Comment:
No problem. I tried to be consistent and used the same syntax already used for access_periods: https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/templates/waypoint/helpers/view.html#L403

Should we change it here too?